draft-ietf-quic-transport.md

```diff
@@ -5018,6 +5034,31 @@ sequence number, or if a sequence number is used for different connection
 IDs, the endpoint MAY treat that receipt as a connection error of type
 PROTOCOL_VIOLATION.
 
+The Retire Prior To field is a requirement for the peer to retire all connection
+IDs with a sequence number less than the specified value.  This includes the
+initial and preferred_address transport parameter connection IDs.  The peer
+SHOULD immediately retire all the connection IDs.
+
+The sender of the NEW_CONNECTION_ID frame MAY remove the connection IDs after 3
+PTO, even if the peer has not retired them yet.  The 3-PTO timer starts on
```

Comment:
I forgot about RETIRE_CONNECTION_ID frame the first time I read this.  Can you say something like: "not retired them with RETIRE_CONNECTION_ID yet."